app/src/main/java/com/yogeshpaliyal/deepr/ui/screens/home/ProfileManagement.kt

```
@@ -0,0 +1,362 @@
+
+                            coroutineScope.launch {
+                                try {
+                                    viewModel.updateProfile(profile.id, trimmed, profile.themeMode, profile.colorTheme)
+                                    onDismiss()
+                                    Toast
+                                        .makeText(
+                                            context,
+                                            context.getString(R.string.profile_updated_successfully),
+                                            Toast.LENGTH_SHORT,
+                                        ).show()
+                                } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)